### PR TITLE
[OCPBUGS#34946] Update ShiftStack apiVersion in example

### PR DESCRIPTION
**Pull request by maxwelldb**
Update an API version per OCPBUGS-34946

Version(s): 4.16

Issue: [OCPBUGS-34946](https://issues.redhat.com/browse/OCPBUGS-34946)

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information: imatza-rh is QE. And yes, I will squash commits before merging.

### CODE REVIEW

File: modules/machineset-yaml-osp.adoc [hunk 1]
```diff
@@ -74,7 +74,7 @@ ifdef::infra[]
 endif::infra[]
       providerSpec:
         value:
-          apiVersion: openstackproviderconfig.openshift.io/v1alpha1
+          apiVersion: machine.openshift.io/v1alpha1
```
Comment (imatza-rh) [inline modules/machineset-yaml-osp.adoc:77]:
```suggestion
          apiVersion: machine.openshift.io/v1alpha1
```

**Comment by maxwelldb** [comment]
@imatza-rh That apiVersion value is present in these live modules, too:

https://github.com/openshift/openshift-docs/blob/main/modules/installation-osp-setting-worker-affinity.adoc
https://github.com/openshift/openshift-docs/blob/main/modules/machineset-yaml-osp-sr-iov-port-security.adoc
https://github.com/openshift/openshift-docs/blob/main/modules/machineset-yaml-osp-sr-iov.adoc

Should this change occur in those places, too?

**Comment by imatza-rh** [comment]
> @imatza-rh That apiVersion value is also present in these live modules:
> 
> https://github.com/openshift/openshift-docs/blob/main/modules/installation-osp-setting-worker-affinity.adoc https://github.com/openshift/openshift-docs/blob/main/modules/machineset-yaml-osp-sr-iov-port-security.adoc https://github.com/openshift/openshift-docs/blob/main/modules/machineset-yaml-osp-sr-iov.adoc
> 
> Should this change occur in those places, too?

I think yes. @pierreprinetti can you please confirm?

**Comment by maxwelldb** [comment]
Made that change assuming you're right. Waiting for @pierreprinetti to confirm.

**Comment by pierreprinetti** [comment]
Apologies, I had missed the notification. Let me ping @mdbooth for this

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by GroceryBoyJr** [comment]
@maxwelldb did you mean to keep the three commits, or can you squash?

**Comment by maxwelldb** [comment]
> @maxwelldb did you mean to keep the three commits, or can you squash?

Will indeed squash prior to merge.

**Pull request merged**